### PR TITLE
SQL: Guild shop prices/quantities update

### PR DESCRIPTION
**Pull request by Hozu**
As requested by @bendangelo, his source is http://pastebin.com/vByTKTye

### CODE REVIEW

**Comment by teschnei** [comment]
what the hell kind of source is that?

**Comment by Hozu** [comment]
No idea, ask him about it.

**Comment by bendangelo** [comment]
I got this from someone. I haven't actually confirmed any of this as being correct, will look into that tomorrow.

File: sql/guild_shops.sql [hunk 1]
```diff
@@ -726,34 +726,34 @@ INSERT INTO `guild_shops` VALUES ('525', '17407', '270', '270', '20', '6', '12')
...
-INSERT INTO `guild_shops` VALUES ('526', '4154', '5250', '13300', '55', '0', '22');
-INSERT INTO `guild_shops` VALUES ('526', '4162', '5250', '13300', '80', '0', '32');
-INSERT INTO `guild_shops` VALUES ('526', '4164', '1050', '6832', '165', '0', '65');
--- INSERT INTO `guild_shops` VALUES ('526', '4165', '900', '5712', '165', '0', '65');
+INSERT INTO `guild_shops` VALUES ('526', '4150', '1945', '4200', '80', '0', '32');
+INSERT INTO `guild_shops` VALUES ('526', '4151', '890', '900', '80', '0', '0');
+INSERT INTO `guild_shops` VALUES ('526', '4154', '20', '30', '1', '0', '0');
+INSERT INTO `guild_shops` VALUES ('526', '4162', '1250', '2300', '80', '0', '32');
+INSERT INTO `guild_shops` VALUES ('526', '4164', '1500', '2400', '165', '0', '0');
+-- INSERT INTO `guild_shops` VALUES ('526', '4165', '1400', '2300', '165', '0', '0');
```
Comment (bendangelo) [inline sql/guild_shops.sql]: This supposed to be commented out?

File: sql/guild_shops.sql [hunk 2]
```diff
@@ -726,34 +726,34 @@ INSERT INTO `guild_shops` VALUES ('525', '17407', '270', '270', '20', '6', '12')
...
 INSERT INTO `guild_shops` VALUES ('526', '4148', '1200', '1377', '80', '0', '32');
-INSERT INTO `guild_shops` VALUES ('526', '4150', '1945', '9549', '80', '0', '32');
-INSERT INTO `guild_shops` VALUES ('526', '4151', '880', '2944', '80', '0', '32');
-INSERT INTO `guild_shops` VALUES ('526', '4154', '5250', '13300', '55', '0', '22');
-INSERT INTO `guild_shops` VALUES ('526', '4162', '5250', '13300', '80', '0', '32');
-INSERT INTO `guild_shops` VALUES ('526', '4164', '1050', '6832', '165', '0', '65');
--- INSERT INTO `guild_shops` VALUES ('526', '4165', '900', '5712', '165', '0', '65');
+INSERT INTO `guild_shops` VALUES ('526', '4150', '1945', '4200', '80', '0', '32');
+INSERT INTO `guild_shops` VALUES ('526', '4151', '890', '900', '80', '0', '0');
+INSERT INTO `guild_shops` VALUES ('526', '4154', '20', '30', '1', '0', '0');
```
Comment (bendangelo) [inline sql/guild_shops.sql]: Keep the price around '5250', '13300'

File: sql/guild_shops.sql [hunk 3]
```diff
@@ -726,34 +726,34 @@ INSERT INTO `guild_shops` VALUES ('525', '17407', '270', '270', '20', '6', '12')
...
+INSERT INTO `guild_shops` VALUES ('526', '4128', '4000', '10000', '30', '0', '12');
 INSERT INTO `guild_shops` VALUES ('526', '4148', '1200', '1377', '80', '0', '32');
-INSERT INTO `guild_shops` VALUES ('526', '4150', '1945', '9549', '80', '0', '32');
-INSERT INTO `guild_shops` VALUES ('526', '4151', '880', '2944', '80', '0', '32');
-INSERT INTO `guild_shops` VALUES ('526', '4154', '5250', '13300', '55', '0', '22');
-INSERT INTO `guild_shops` VALUES ('526', '4162', '5250', '13300', '80', '0', '32');
-INSERT INTO `guild_shops` VALUES ('526', '4164', '1050', '6832', '165', '0', '65');
--- INSERT INTO `guild_shops` VALUES ('526', '4165', '900', '5712', '165', '0', '65');
+INSERT INTO `guild_shops` VALUES ('526', '4150', '1945', '4200', '80', '0', '32');
+INSERT INTO `guild_shops` VALUES ('526', '4151', '890', '900', '80', '0', '0');
```
Comment (bendangelo) [inline sql/guild_shops.sql]: Revert this as well

File: sql/guild_shops.sql [hunk 4]
```diff
@@ -726,34 +726,34 @@ INSERT INTO `guild_shops` VALUES ('525', '17407', '270', '270', '20', '6', '12')
...
+INSERT INTO `guild_shops` VALUES ('526', '4154', '20', '30', '1', '0', '0');
+INSERT INTO `guild_shops` VALUES ('526', '4162', '1250', '2300', '80', '0', '32');
```
Comment (bendangelo) [inline sql/guild_shops.sql]: Revert

File: sql/guild_shops.sql [hunk 5]
```diff
@@ -726,34 +726,34 @@ INSERT INTO `guild_shops` VALUES ('525', '17407', '270', '270', '20', '6', '12')
...
+INSERT INTO `guild_shops` VALUES ('526', '4164', '1500', '2400', '165', '0', '0');
```
Comment (bendangelo) [inline sql/guild_shops.sql]: revert

File: sql/guild_shops.sql (hunk 1 above)
Comment (Hozu) [inline sql/guild_shops.sql]: That one is a mistake, sorry.

**Comment by Hozu** [comment]
Ok fixed the one commented line (was guild id 5262 not 526...) and reverted the ones you pointed out.

**Comment by TeoTwawki** [comment]
FAIL: I was viewing individual commit link and not the merge page..So never saw anyones conversation here.

**Comment by bendangelo** [comment]
I looked up the values on ffxi wiki and the changes seemed reasonable. Feel free to call anything out.

**Comment by TeoTwawki** [comment]
@bendangelo apologies, had I clicked the correct link and/or payed better attention to what page I landed on, I would've kept my mouth shut because I'd have seen where you commented on diffs indicating you had already been checking on it. All I had read was the final commit+pastebin.

**Comment by bendangelo** [comment]
Cool no problem. If you see anything that seems wrong just let me know.

**Pull request merged**